templates/desktop/statistics.html

```diff
@@ -33,109 +33,138 @@ <h1>Ubuntu user statistics</h1>
+      <div class="col-12 u-align--center">
+        <svg id="os-architecture"></svg>
       </div>
-      <p class="u-no-margin--bottom">Firmware initialises hardware components and the operating system at start up.</p>
     </div>
-    <div class="col-4 p-card--flex">
-      <header class="p-card__header">
-        <h3 class="p-heading--four">Display server</h3>
-      </header>
-      <div id="display-server" class="p-card__center-content"></div>
-      <p class="u-no-margin--bottom">The display server controls the input and output of its clients to and from the rest of the operating system and the hardware.</p>
+    <div class="col-6 p-divider__block">
+      <div class="col-12">
+        <h2 class="p-heading--four">Display server</h2>
+        <p>The display server controls the input and output of its clients to and from the rest of the operating system and the hardware.</p>
+      </div>
+      <div class="col-12 u-align--center">
+        <svg id="display-server"></svg>
+      </div>
     </div>
   </div>
 </section>
-<section class="p-strip is-shallow is-bordered">
+<section class="p-strip--light is-bordered">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-6">
+      <img src="https://via.placeholder.com/350x150" />
+    </div>
+    <div class="col-6">
+        <h3 class="p-heading--four">Firmware</h3>
+        <p>The display server controls the input and output of its clients to and from the rest of the operating system and the hardware.</p>
+    </div>
+  </div>
+</section>
+<section id="user-report" class="p-strip is-shallow is-bordered">
   <div class="row">
-    <h3>What graphics setup do users have?</h2>
+    <div class="col-6">
+      <h3>What graphics setup do users have?</h3>
+      <p>*Data for graphics setup only applies to physical machines</p>
+    </div>
   </div>
-  <div class="row u-equal-height">
-    <div class="col-6 prefix-2">
-      <div class="p-card--flex">
-        <div id="one-screen" class="p-card__center-content"></div>
-        <p class="u-no-margin--bottom">This was detected on install so many users may have connected screens after the installing Ubuntu.</p>
+  <div class="row u-equal-height u-vertically-center p-divider">
+    <div class="col-6 p-divider__block">
+      <div class="row">
+        <div class="col-12 u-align--center">
+          <img src="https://via.placeholder.com/350x150" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-12">
+          <p>*On install. This was detected on install so many users may have connected screens after the installing Ubuntu.</p>
+        </div>
       </div>
     </div>
-    <div class="col-6 suffix-2">
-      <div class="p-card--flex">
-        <div id="one-gpu" class="p-card__center-content"></div>
-        <p class="u-no-margin--bottom">Graphical Processing Unit (GPU) is used to generate videos and images for displaying on screen.</p>
+    <div class="col-6 p-divider__block">
+      <div class="row">
```

Comment:
There is a parent row at line https://github.com/canonical-websites/www.ubuntu.com/pull/4135/files#diff-e60ec2befd8be52ae99dcdc29c3bb7a2R145. You shouldn't need to nest them here.